docs/assets/js/docs.js

```diff
@@ -0,0 +1,16 @@
+m_triggers = document.querySelectorAll("[data-trigger-modal]");
+
+for(var i=0; i < m_triggers.length; i++) {
+  mt = m_triggers[i];
+
+  mt.addEventListener("click", function(e) {
+    modal = document.querySelector(this.getAttribute("data-trigger-modal"));
+
+    if(!modal.classList.contains("is-with-backdrop"))
+      modal.style.top = document.scrollingElement.scrollTop + "px";
+
+    modal.classList.toggle("is-active");
+
+    e.preventDefault();
+  })
+}
```

Comment:
This whole section needs revision to match up to the JS standards. Specifically:

 * Declare variables as `const`, not `var` (L3) or nothing (L1)
 * Spaces: after `for`, around `=`, after `function`
 * Variable names are `camelCase` not `snake_case`
 * Missing braces around single-line `if` statement body
 * Missing semicolon after the `addEventListener` call
 * Recommend using descriptive names (what are `m_triggers` and `mt`?)

Reply:
Oh yeah, sorry. Will do later today/tomorrow. I think the same applies to the JS in #6, so no need to review-critizise it there, until fixed here.

Reply:
Should be fixed now. Please re-review. :)